### DISCUSSION TITLE
My Account Was Flagged For No Reason？？？

### DISCUSSION THREAD

**Discussion by rockenwu** [community/community#18560] [Other Feature Feedback, Questions, & Ideas]
My Account Was Flagged For No Reason？？？Help me，PLs.
![微信截图_20220614160701](https://user-images.githubusercontent.com/107275008/173527134-7f5f2c18-e417-4278-b177-192a1caa2f9a.png)

**Comment by Zi-Tao** [comment]
@rockenwu https://webapps.stackexchange.com/questions/105956/my-github-account-has-been-suddenly-flagged-and-hidden-from-public-view-how

**Comment by D7EAD** [comment]
Hi!

Sometimes Github's spam-hunting bots make mistakes. If your account has been flagged in error, kindly submit a support ticket to Github's Support Engineers and they will swiftly correct the mistake. You can do so [here](https://support.github.com/contact/account).
